### DISCUSSION TITLE
Seeburg jukebox wallbox decoder to MQTT via Tasmota

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#15650] [Show and tell]
For many years now, I have had a working Seeburg 3WA-200 in my kid's den, connected to an iPod. This allows them and their friends to play whatever they wanted... but as time goes on, they grew up and moved away... (hard to believe...)

So it has been sitting unused for years... As I keep seeing it unused, I keep thinking about what to do with it... Well, Tasmota came to the rescue....

Code at:

~~~
https://github.com/trlafleur/Tasmota_Seeburg_3WA_3W1_Decoder
~~~

**Comment by arendst** [comment]
Nice.

**Comment by necro-nemesis** [comment]
Interesting project. I came across your repo looking for a way to decode and then re-encode Seeburg wallbox pulses. Now you may ask why on earth would anyone want to do this? Well I wanted to use an SC1 wallbox due to it having built in speakers on a Seeburg 222 jukebox and found out the hard way Seeburg reversed it's previous encoding of letters first and numbers second to the other way around. A decoder/encoder that could in turn pulse the 24VAC supply back out to a jukebox would make wallboxes interchangeable, possibly even between different manufacturers.

**Comment by trlafleur** [comment]
all the parts are there to do what you want....


~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur
